kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -1,6 +1,18 @@
 <template>
 
   <div>
+    <KRouterLink
+      appearance="raised-button"
+      :to="{ path: 'new/123/edit' }"
+      text="Edit section"
+    />
+
+    <KRouterLink
+      appearance="raised-button"
+      :to="{ path: 'new/123/replace-questions' }"
```

Comment:
I'm not sure what these paths should be based on the architectural decisions made with @nucleogenesis, but I think if it's possible for these to be the real path (or alternatively, maybe they should be just a button and not a router link at all), it would be worth trying to work that out now and get as much of the behavior in place as possible, even if the actual "saving" and "replacing" to the database doesn't happen until later